xdsl/interactive/app.py

```diff
@@ -126,6 +126,11 @@ class InputApp(App[None]):
     text areas.
     """
 
+    current_argument_pass: tuple[tuple[type[ModulePass], PipelinePassSpec], ...] = ()
```

Comment:
Why do we need this new variable? Can we not just get the last element of the pass pipeline?

Reply:
I explained in the description above - 

"Since functions to do with screens cannot return the variable (i.e the add_pass_with_arguments_to_pass_pipeline), I create a global current_argument_pass variable that holds the variable so that it can be accessed later."


- I think that we do not necessarily want to add this to the pass pipeline in the future when doing multi-level-traversal. no?

Reply:
I don't understand what you mean here

Reply:
Okay I have removed it. Is this better?